config/routes.rb

```diff
@@ -193,6 +193,8 @@
 
   get "contact-admin", to: "support#admin"
 
+  get "mentions_legales", to: "legal_notice#index"
```

Comment:
```suggestion
  get "mentions-legales", to: "legal_notice#index"
```
(pas d'underscore dans les urls)